filebeat/harvester/harvester.go

```diff
@@ -17,12 +17,15 @@ import (
 	"fmt"
 	"regexp"
 
+	"github.com/satori/go.uuid"
+
 	"github.com/elastic/beats/filebeat/config"
 	"github.com/elastic/beats/filebeat/harvester/encoding"
 	"github.com/elastic/beats/filebeat/input"
 )
 
 type Harvester struct {
+	Id                 uuid.UUID
```

Comment:
If we need a ID to track an ephemeral harvester then why not just just assign ID's based on one up counter? There's no requirement for global uniqueness here as far as I can tell.